src/lib/components/event/event-history-timeline.svelte

```diff
@@ -478,72 +504,72 @@
   }
 
   :global(.vis-item.vis-range.ContinuedAsNew) {
-    background-color: #e4e4e7;
-    border-color: #18181b;
+    background-color: theme(colors.purple.300);
+    border-color: theme(colors.purple.300);
     border-radius: 9999px;
     border-width: 2px;
-    color: #17172c;
+    color: theme(colors.black);
   }
 
   :global(.vis-item.vis-point.ContinuedAsNew) {
-    color: #18181b;
+    color: theme(colors.purple.300);
   }
 
   :global(.vis-item.vis-range.Completed, .vis-item.vis-range.Open) {
-    background-color: #dcfce7;
-    border-color: #15803d;
+    background-color: #71d375;
+    border-color: #71d375;
```

Comment:
Could we use a theme color here? cc @Alex-Tideman

Reply:
That `#71d375` is actually a special green (from our brand colors I believe) and not from Tailwind. We're using it in `workflow-status` as well. Looks like we'll be changing it in the near future though, otherwise I would just add it in our tailwind config.